lib/mindwendel_web/controllers/static_page_controller.ex

```diff
@@ -21,4 +21,29 @@ defmodule MindwendelWeb.StaticPageController do
       form: form
     )
   end
+
+  def legal(conn, _params) do
+    if Application.fetch_env!(:mindwendel, :options)[:feature_privacy_imprint_enabled] do
```

Comment:
You should be able to access it with the `.syntax` (`.feature_privacy_imprint...`) as they key should always be there and in these circumstances `.` access is preferred